sdk/analytics/src/main/java/com/klaviyo/analytics/model/Keyword.kt

```diff
@@ -0,0 +1,10 @@
+package com.klaviyo.analytics.model
```

Comment:
Also could be in `core` maybe since its a pretty abstract class?

Reply:
Done

Reply:
Darn, actually can't move this. Weirdly, there's `infix` methods that map back to the base class and so if the base class isn't public, you can't do something like `mapOf(ProfileKey.EMAIL to "emailaddress") because the infix function `to` isn't public!!

Reply:
Oof thats unexpected!

Reply:
Yeah I did not see that coming!